src/meshapi/views/forms.py

```diff
@@ -255,6 +293,48 @@ class NetworkNumberAssignmentRequest:
     install_number: int
 
 
+class NetworkNumberAssignmentRequestSerializer(DataclassSerializer):
+    class Meta:
+        dataclass = NetworkNumberAssignmentRequest
+
+    password = serializers.CharField()
+
+
+nn_form_success_schema = inline_serializer(
+    "NNFormSuccessResponse",
+    fields={
+        "detail": serializers.CharField(),
+        "building_id": serializers.IntegerField(),
+        "install_number": serializers.IntegerField(),
+        "network_number": serializers.IntegerField(),
+        "created": serializers.BooleanField(),
+    },
+)
+
+
+@extend_schema_view(
+    post=extend_schema(
+        tags=["User Forms"],
+        summary="Assign a network number to a given Install object. Used by the NN Assignment form",
```

Comment:
Worthwhile to link to source code and/or meshforms here?

Reply:
I'm nervous to link stuff since our URLs are almost certain to change in the future